src/mon/MgrStatMonitor.cc

```diff
@@ -119,6 +130,18 @@ void MgrStatMonitor::clear_pool_availability(int64_t poolid)
   dout(20) << __func__ << " cleared availability score for pool: " << poolid << dendl;
 }
 
+bool MgrStatMonitor::should_calc_pool_availability() 
+{
+  dout(20) << __func__ << dendl;
+  std::scoped_lock l(lock); 
+
+  utime_t now = ceph_clock_now();
```

Comment:
utime_t now = ceph_clock_now();

since it returns `utime_t` other places that call `ceph_clock_now()` should also return `utime_t`